v03_pipeline/lib/reference_data/config.py

```diff
@@ -372,10 +372,12 @@ def custom_mpc_select(ht):
     },
     'gnomad_qc': {
         '37': {
+            'version': 'v2',
             'custom_import': import_matrix_table,
             'source_path': 'gs://gnomad/sample_qc/mt/gnomad.joint.high_callrate_common_biallelic_snps.pruned.mt',
         },
         '38': {
+            'version': 'v3.1',
```

Comment:
`gnomad_qc` is the only dataset without a version in either the config or the source table, so I created a version field for them here in the config so that the globals validation works for this dataset (`clinvar` and `gnomad_genomes` have versions)

Reply:
👍